### PR TITLE
Limit scope to what we actually use

### PR DESCRIPTION
**Pull request by DeviaVir**
Also ran `go fmt`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,7 +19,7 @@ GOVERSION := 1.9.3
 PROJECT := $(CURRENT_DIR:$(GOPATH)/src/%=%)
 OWNER := $(notdir $(patsubst %/,%,$(dir $(PROJECT))))
 NAME := $(notdir $(PROJECT))
-VERSION := 0.1.3
+VERSION := 0.1.4
```
Comment (mathyourlife-fitbit) [inline Makefile]: 0.1.4 already exists as a release.  This supposed to be 0.1.5?

File: gsuite/config.go [hunk 2]
```diff
@@ -6,27 +6,20 @@ import (
...
 	"strings"
-	"encoding/json"
 )
 
 var defaultOauthScopes = []string{
-	directory.AdminDirectoryCustomerScope,
 	directory.AdminDirectoryGroupScope,
-	directory.AdminDirectoryGroupMemberScope,
-	directory.AdminDirectoryOrgunitScope,
-	directory.AdminDirectoryUserScope,
```
Comment (mathyourlife-fitbit) [inline gsuite/config.go]: Don't need AdminDirectoryUserScope scope for resources like `gsuite_user`?

Comment (DeviaVir) [reply gsuite/config.go]: I thought this was tested and not needed, I just verified it myself and yes we do need the UserScope for `gsuite_user`.  Good catch!

File: Makefile (hunk 1 above)
Comment (DeviaVir) [reply Makefile]: Yeah, updated it.

**Comment by mathyourlife-fitbit** [comment]
👍

**Pull request merged**